template.yaml

```
@@ -934,6 +962,299 @@ Resources:
             Unit: Count
   #endregion
 
+  #region Authentication Callback Lambda
+
+  AuthenticationCallbackFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_116: DLQ is not appropriate for a Lambda invoked by an API
+    # checkov:skip=CKV_AWS_117: VPC settings are global
+    Properties:
+      FunctionName: !Sub ${Environment}-AuthenticationCallbackFunction
+      AutoPublishAlias: latest
+      CodeUri: ./oidc-api
+      Handler: uk.gov.di.authentication.oidc.lambda.AuthenticationCallbackHandler::handleRequest
+      ReservedConcurrentExecutions: 1
+      LoggingConfig:
+        LogGroup: !Ref AuthenticationCallbackFunctionLogGroup
+      Environment:
+        Variables:
+          # checkov:skip=CKV_AWS_173: These environment variables do not require encryption.
+          ENVIRONMENT: !Sub ${Environment}
+          ACCOUNT_INTERVENTION_SERVICE_ACTION_ENABLED: true
+          ACCOUNT_INTERVENTION_SERVICE_CALL_ENABLED: true
+          ACCOUNT_INTERVENTION_SERVICE_URI: !Ref AccountInterventionsServiceUriSecret
```

```suggestion
          ACCOUNT_INTERVENTION_SERVICE_URI:  !Sub
            - '{{resolve:secretsmanager:${SecretArn}:SecretString}}'
            - SecretArn: !Ref AccountInterventionsServiceUriSecret
```

I've realised the old one doesn't work, my bad 😁